klips-api/example_requests/send-geotiff-dresden.json

```diff
@@ -3,7 +3,7 @@
     "source": "hhi",
     "email": "info@example.de",
     "payload": {
-        "url": "http://nginx/ecostress_3035_float32.tif",
+        "url": "http://nginx/sample_dresden_float32_2bands.tif",
```

Comment:
Aint that `http://nginx:81` in the future? Or should the adaption be part of #115 ?

Reply:
@chrismayer true. This will change. But I intentionally left to be fixed in #115